python/plugins/MetaSearch/dialogs/maindialog.py

```diff
@@ -84,8 +84,10 @@ def __init__(self, iface):
         self.catalog_password = None
         self.context = StaticContext()
 
-        version = self.context.metadata.get('general', 'version')
-        self.setWindowTitle(self.tr('MetaSearch {0}').format(version))
```

Comment:
is including the version as part of the window title a version?  I've always found this valuable in helping debug.

Reply:
Well, now that metasearch is core the version number follows qgis versioning. So basically it's redundant/misleading information now. (Note that we also removed the version number from the main qgis windows in 3.0, based on a survey of other applications we found that barely any modern apps have the version in the title bar. The thinking here is that it's information only of interest to a small subset of users, and they can find this out via the about screen).

Reply:
Thanks @nyalldawson.  Makes sense.  Merging.